docs/topics/forms/descriptions/content-description.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Description has text only
+layout: default
+parent: Input and description
+nav_order: 3
+has_video: true
+---
+
+# Only include text in the description
+
+The HTML attribute `aria-describedby`, which you use to connect a description, must only contain "plain text". Plain text represents text without further formatting and has no additional HTML for formatting or meaning. Link lists, headings or tooltips.
+
+We call formatted text "rich text". Screen readers don't convey this information correctly or even skip it entirely in a description.
+
+## Demo of a description with rich text announced with VoiceOver in Safari
+
+<video data-able-player data-youtube-nocookie="true" data-youtube-id="6brKmJfDtNQ" data-heading-level="0"></video>
+
+In this [demo video](https://www.youtube.com/watch?v=6brKmJfDtNQ), VoiceOver in Safari 18.6 skips the bulleted list from the description when a form field receives keyboard focus. NVDA 2024.4.1 in Firefox does the same. The codepen with the HTML/CSS of this demo: [Demo rich text in a description](https://codepen.io/rianrietveld/pen/emJedVR).
+
+## Description decision tree
+
+What are the options for adding information and explanations to a form and its input fields? Follow the decision tree in 6 questions.
+
+### Question 1: Does the information apply to all form fields?
+
+**Yes**: Place the information just above the form and below its heading. You can use rich text here.  
```

Comment:
Should `form` be shown as `<form>`, to make it explicit that it should be outside the form element?